.bitmap

```diff
@@ -777,6 +777,12 @@
         "mainFile": "index.ts",
         "rootDir": "scopes/toolbox/string/strip-trailing-char"
     },
+    "test-task": {
```

Comment:
@giteden don't use this as component name

Reply:
removed it